configure/CMakeLists.txt

```diff
@@ -151,6 +151,16 @@ Please install cppzmq (https://github.com/zeromq/cppzmq) on your system and/or u
 <OMNI_BASE>/include and system include directories).")
 endif()
 
+TRY_COMPILE(HAS_COMPATIBLE_ZMQ_HPP_VERSION
+  ${CMAKE_BINARY_DIR}/configure
+  SOURCES ${CMAKE_SOURCE_DIR}/configure/test_zmq_hpp.cpp
+  CMAKE_FLAGS "-I ${CPPZMQ_BASE}/include -I ${ZMQ_PKG_INCLUDE_DIRS}/include"
+  LINK_LIBRARIES ${ZMQ_PKG_LIBRARIES})
```

Comment:
I guess we need to avoid `${ZMQ_PKG_XXX}` here and use `${CPPZMQ_BASE}` everywhere. As it is now - cmake compiles against cppzmq from pkg-config (from `apt-get`) which is indeed incompatible on Debian 9&7